src/models/migration.js

```diff
@@ -22,18 +22,40 @@ export default async () => {
         phone VARCHAR(15) not null,
         password TEXT not null
         )`
-      ).catch(error => { throw error });
+      ).then(async () => {
+        await query(`INSERT INTO users(
+        id, name, email, phone, password
+        ) VALUES(
+        '5d5f3ead-f5e8-41df-b997-a71171506f48',
+        'Young',
+        'young@gmail.com',
+        '08013562789',
+        '$2b$10$HI/qDHmOIQruybpEpEKPX.gIGKm.DzhLa44Y/Hm/JMAq1ia/RLSOW'
+        )`);
+      }).catch(error => {
+        throw error
+      });
   await query(`CREATE TABLE IF NOT EXISTS foods(
         id UUID PRIMARY KEY DEFAULT gen_random_uuid(),
         name VARCHAR(100) not null,
         description TEXT not null,
         image TEXT, price MONEY not null
         )`
-      ).catch(error => { throw error });
+      ).then(async () => {
+        await query(`INSERT INTO foods(
```

Comment:
Same for this. I don't think your migration files should insert any data into your database.

Reply:
I thought you said you agreed I could leave this until the endpoint is created